Configuration/Geometry/README.md

```diff
@@ -18,12 +18,15 @@ Calorimeters:
 * C8: HGCal (v10 post TDR HGCal Geometry) + Phase2 HCAL and EB + Tracker cables in calorimeter region
 * C9: HGCal (v11 post TDR HGCal Geometry same as C8 + corner centering for HE part)
 * C10: HGCal (as in C9) + HFNose with corrected wafer size + Phase2 HCAL and EB
+* C11: HGCal (v12 post TDR HGCal Geometry same as C9 + modified support structure + full list of masked wafers)
+* C12: HGCal (as in C11) + HFNose with corrected wafer size + Phase2 HCAL and EB
 
 Muon system:
 * M2: Phase2 muon system for TDR w/ GE2/1, ME0, RE3/1, RE4/1 (incl. granularity in ME0, staggered GE2/1)
 * M3: same as M2 with change to the number of iRPC strips from 192 to 96 as in TDR
 * M4: same as M3 but no Overlaps inside the Muon System and MB4Shields completed
 * M5: same as M4 but with: the right value for YE3 size, no "hidden" overlaps inside the Muon System and iRPC updated.
+* M6: same as M4 but with adjustment of ME0 in view of updated boundaries
```

Comment:
should this descend from M5 rather than M4?

Reply:
It has indeed and I updated that